usaspending_api/api_contracts/contracts/v2/search/spending_by_subaward_grouped.md

```diff
@@ -0,0 +1,258 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending by Award [/api/v2/search/spending_by_subaward_grouped/]
+
+This endpoint supports the advanced search page and allow for complex filtering for specific subsets of spending data.
+
+## POST
+
+Searches for subaward records based on a provided set of filters and groups them by their prime award.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `filters` (required, AdvancedFilterObject)
+            These filters are applied at the Subaward level (`children`). They will not affect the Prime Awards (`results`) themselves
+        + `fields` (required, SpendingBySubawardFields)
+            The field names to include for `children` objects (Subawards) the response.
```

Comment:
Is there a typo?  Should this be "The field names to include for `children` objects (Subawards) **in** the response."

Reply:
Yes, thanks Andrea! Fixing now